inc/comments-template.php

```diff
@@ -13,7 +13,8 @@
 	<div id="coral_talk_<?php echo absint( $rand ); ?>"></div>
 	<script src="<?php echo esc_url( $talk_url . '/embed.js' ); ?>" async onload="
 		Coral.Talk.render(document.getElementById('coral_talk_<?php echo absint( $rand ); ?>'), {
-			talk: '<?php echo esc_url( $talk_url ); ?>'
+			talk: '<?php echo esc_url( $talk_url ); ?>',
+			asset_id: '<?php echo esc_js( coral_talk_get_asset_id() ); ?>'
```

Comment:
`wp_json_encode()` wraps in double quotes, which break the `onload` attribute that this needs to be inside of per Coral spec